test/evaluation/submissionRequirements.spec.ts

```diff
@@ -13,180 +13,176 @@ const HOLDER_DID = 'did:example:ebfeb1f712ebc6f1c276e12ec21';
 describe('Submission requirements tests', () => {
   it('Evaluate submission requirements all from group A', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.